rocky/reports/templates/partials/export_report_settings.html

```diff
@@ -13,115 +12,65 @@ <h2>{% translate "Report name" %}</h2>
-                                                {% translate "Day" %} ({{ current_datetime|date:"N jS Y, H:i" }})
-                                            </option>
-                                            <option value="{% translate "Week" %} {{ current_datetime|date:"W, Y" }}">
-                                                {% translate "Week" %} ({% translate "Week" %} {{ current_datetime|date:"W, Y" }})
-                                            </option>
-                                            <option value="{{ current_datetime|date:"N, Y" }}">
-                                                {% translate "Month" %} ({{ current_datetime|date:"N, Y" }})
-                                            </option>
-                                            <option value="{{ current_datetime|date:"Y" }}">{% translate "Year" %} ({{ current_datetime|date:"Y" }})</option>
+                                            <option value="%b %d %Y, %H:%M">{% translate "Day" %} ({{ current_datetime|date:"N jS Y, H:i" }})</option>
+                                            <option value="week">{% translate "Week" %} ({% translate "Week" %} {{ current_datetime|date:"W, Y" }})</option>
+                                            <option value="%b, %Y">{% translate "Month" %} ({{ current_datetime|date:"N, Y" }})</option>
+                                            <option value="%Y">{% translate "Year" %} ({{ current_datetime|date:"Y" }})</option>
                                         </select>
-                                    </div>
-                                </fieldset>
-                            {% endfill %}
-                            {% fill "footer_buttons" %}
-                            <button type="button" class="confirm-button confirm-modal-button">{% translate "Confirm" %}</button>
-                            <button type="button" class="ghost close-modal-button">{% translate "Cancel" %}</button>
-                        {% endfill %}
-                    {% endcomponent %}
-                    {% component_css_dependencies %}
-                </td>
-            </tr>
-        {% endwith %}
-    {% endfor %}
-</tbody>
-</table>
-{% csrf_token %}
-<button type="submit" form="generate_report">
-    {% include "forms/report_form_fields.html" %}
+                                    </td>
+                                </tr>
+                            {% endwith %}
+                        {% endfor %}
+                    </tbody>
+                </table>
+                {% csrf_token %}
+                <button type="submit" form="generate_report">
+                    {% include "forms/report_form_fields.html" %}
 
-    {% translate "Generate report" %}<span class="icon ti-chevron-right"></span>
-</button>
-</form>
-{% else %}
-{% include "partials/return_button.html" with btn_text="Go back" %}
+                    {% translate "Generate report" %}<span class="icon ti-chevron-right"></span>
```

Comment:
This icon should get an `aria-label` or `aria-hidden="true"` - probably the latter, as there is already accessible text on the button.